pkgs/data/icons/catppuccin-papirus-folders/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, papirus-icon-theme
+, gtk3
+, bash
+, getent
+, flavor ? "mocha"
+, accent ? "blue"
+, ...
+}:
+let
+  validAccents = [ "blue" "flamingo" "green" "lavender" "maroon" "mauve" "peach" "pink" "red" "rosewater" "sapphire" "sky" "teal" "yellow" ];
+  validFlavors = [ "latte" "frappe" "macchiato" "mocha" ];
+  pname = "catppuccin-papirus-folders";
+in
+lib.checkListOfEnum "${pname}: accent color" validAccents [ accent ]
+  lib.checkListOfEnum "${pname}: flavor"
+  validFlavors [ flavor ]
+  stdenvNoCC.mkDerivation
+rec {
+  inherit pname;
+  version = "unstable-2022-12-04";
+
+  src = fetchFromGitHub {
+    owner = "catppuccin";
+    repo = "papirus-folders";
+    rev = "1a367642df9cf340770bd7097fbe85b9cea65bcb";
+    sha256 = "sha256-mFDfRVDA9WyriyFVzsI7iqmPopN56z54FvLkZDS2Dv8=";
+  };
+
+  buildInputs = [ bash getent gtk3 papirus-icon-theme ];
```

```suggestion
  nativeBuildInputs = [ gtk3 ];
```

The only build input that actually gets used is `gtk3` but it's used as a native build input for `gtk-update-icon-cache`